R/commonDiscoverDistributions.R

```diff
@@ -586,27 +616,35 @@
   return(res)
 }
 
-.ldFillFitStatisticsTable <- function(table, results, options, ready){
+.ldFillFitStatisticsTable <- function(table, results, options, ready, normality=FALSE){
   if(!ready) return()
   if(is.null(results)) return()
   if(is.null(table)) return()
 
 
-  allTests <- c("kolmogorovSmirnov", "cramerVonMisses", "andersonDarling", "shapiroWilk", "chiSquare")
+  allTests <- c("kolmogorovSmirnov", "cramerVonMisses", "andersonDarling", "lillienfors", "shapiroWilk", "chiSquare")
   tests <- allTests[allTests %in% names(options)]
   testNames <- c(gettext("Kolmogorov-Smirnov"),
                  gettext("Cramér-von Mises"),
                  gettext("Anderson-Darling"),
+                 gettext("Lillienfors"),
                  gettext("Shapiro-Wilk"),
                  gettext("Chi-square"))[allTests %in% names(options)]
 
   whichTests <- unlist(options[tests])
 
   results$test <- testNames
+  rownames(results) <- tests
   res <- results[whichTests,]
 
   table$setData(res)
 
+  if (!normality && (isTRUE(options[["cramerVonMisses"]]) || isTRUE(options[["andersonDarling"]]))) {
```

Comment:
is `isTRUE` really necessary here?

Reply:
Yes, because for some distributions this option is not available (the option is NULL)

And yes it is horrible that I implemented it that way in the first place, I should do a dispatch based on distribution classes, but that requires a larger refactor which is not a focus in this PR